DnD-Montreal/session-tome
# Redirect to campaign detail upon entry creation

### Description
Closes #500 

Once a  **campaign**  entry or dm entry is successfully created, the user is now redirected to the campaign detail page instead of the character detail or dm entry index pages respectively.

To test:
1. Seed database
2. Create a dm entry for a campaign the signed in user is dm for
(if they are not dm for any campaign, create a campaign and assign them dm)
3. Observe that you have been redirected to campaign detail page
4. Create an entry for a campaign the signed in user is participating in with a character
5. Observe that you have been redirected to campaign detail page
